src/Factory.php

```diff
@@ -7,7 +7,9 @@ class Factory
     public static function create()
     {
         // @codeCoverageIgnoreStart
-        if (class_exists('libev\EventLoop', false)) {
+        if (function_exists('uv_default_loop')) {
+            return new LibUvLoop();
```

Comment:
Drop the `()` for consistency.

Reply:
I'd rather keep this as-is and eventually address the other two as CS violations :+1:

Reply:
Lets fix this in a follow up PR as it's 2 vs. 2 right now for both styles